components/common/src/main/java/com/hotels/styx/common/StateMachine.java

```diff
@@ -203,5 +203,11 @@ public int hashCode() {
         public StateMachine<S> build() {
             return new StateMachine<>(initialState, stateEventHandlers, inappropriateEventHandler, stateChangeListener);
         }
+
```

Comment:
Hey David. 

I would just leave this in. When debugging unit tests or e2e tests I can just add a `.debugTransitions` call to the state machine builder, and remove when ready.

Of course it could be integrated more nicely. 🤔